pkg/egressgateway/manager_privileged_test.go

```diff
@@ -653,7 +747,7 @@ func waitForReconciliationRun(tb testing.TB, egressGatewayManager *Manager, curr
 			return count
 		}
 
-		time.Sleep(10 * time.Millisecond)
+		time.Sleep(30 * time.Millisecond) // worried that I need to change this, what's the huge performance blocker here?
```

Comment:
Remembered this when addressing some feedback, and what to highlight since it can easily get lost. I'm still a bit confused why I had to significantly raise the timeout. Anyone has a theory as to why? cc @julianwiedmann

Reply:
I've seen this as well in the past, when experimenting with map migration for the EGW policy map (effectively managing two policy maps in parallel). We now reconcile two maps, so the total reconciliation takes longer. I don't have a good feeling whether the extra time is expected / excessive ...

Reply:
Made a TODO comment to investigate why the timeout increase is necessary.